templates/index.mustache

```diff
@@ -19,25 +19,25 @@
             Start-ups, Design
             Thinker, Agile und Lean Startup-Experten der Metropolregion Frankfurt Rhein Main zusammen.</p>
 
-        <a class="button turquoise with-arrow" target="_self" href="tickets">Jetzt Ticket kaufen</a>
+        <a class="button yellow big" target="_self" href="tickets">Jetzt Ticket kaufen</a>
```

Comment:
Why did you remove the icons? Jens won't like that change... 😏

Reply:
It's not that I don´t like them :) I took them out because the headline was flowing around the icon in a way that made the text less readable. 

Happy to take them back in if Jens comlains :) but would then suggest to put them on top of the headline

Reply:
Maybe a topic for today's call?